addons/account/views/account_move_views.xml

```diff
@@ -848,11 +848,11 @@
                                 </div>
 
                                 <label for="journal_id"
-                                       groups="account.group_account_readonly"
+                                       groups="account.group_account_invoice"
```

Comment:
Why?  Someone in readonly should see the journals as well I suppose.

Reply:
@jco-odoo we had to change this as group_account_readonly is not present when only the invoicing app is installed. Is there a readonly corresponding group in invoicing that I can use instead?

Reply:
@leonardopavanrocha Is not it defined in the account module?  It does not seem to have anything to do with this PR.

Reply:
Yes, it's defined but for some reason the invoicing manager is not in this group when only invoicing is installed. This "bug" was discovered when testing a fix done in the enterprise branch.